packages/expo-updates/ios/EXUpdates/AppLoader/FileDownloader.swift

```diff
@@ -123,6 +123,20 @@ internal final class FileDownloader: NSObject, URLSessionDataDelegate {
       fromURL: url,
       extraHeaders: extraHeaders
     ) { data, response in
+      guard let data = data else {
+        let errorMessage = String(
+          format: "File download response was empty for URL: %@",
+          [url.absoluteString]
+        )
+        self.logger.error(message: errorMessage, code: UpdatesErrorCode.assetsFailedToLoad)
+        errorBlock(NSError(
+          domain: ErrorDomain,
+          code: FileDownloaderErrorCode.InvalidResponseError.rawValue,
+          userInfo: [NSLocalizedDescriptionKey: errorMessage]
+        ))
+        return
+      }
```

Comment:
I'm not sure this is an error: if we get a 204, it's just an empty file (`touch empty.txt`)

Reply:
Makes sense